packages/vanchor-client/src/index.ts

```diff
@@ -0,0 +1,11 @@
+import { GetTotalValueLocked } from './queries/SchieldedTransactions'
+
+async function main() {
+    const value: number = await GetTotalValueLocked();
```

Comment:
The value returned here, for which chain? and for which VAnchor?

Reply:
Not 100% aware of how this value is computed, and the requirement of the spec; However, afaik the shieldedTransactions values to get the TVL should not be computed on client side by query all the 1000s of shieldedTransactions and combine their values together. A better approach here, is to have a custom value for each VAnchor, call it `VAnchorTVL` for example, and whenever we see a new shielded tx, we increment/decrement the TVL by the value of shielded tx value. this way it is just one shot query to the graph service.

Reply:
Resolved.